src/test/scala/org/apache/spark/sql/IndexSuite.scala

```diff
@@ -22,17 +22,18 @@ import org.apache.hadoop.fs.Path
 
 import org.apache.spark.sql.internal.IndexConf._
 import org.apache.spark.sql.functions.{col, lit}
+import org.apache.spark.sql.types._
 
 import com.github.lightcopy.implicits._
 import com.github.lightcopy.testutil.{SparkLocal, UnitTestSuite}
 import com.github.lightcopy.testutil.implicits._
 
 class IndexSuite extends UnitTestSuite with SparkLocal {
-  override def beforeAll {
+  before {
```

Comment:
Should add comment saying that we need to reset `SparkSession` for tests otherwise it is cached in `Metastore`.